sdk/template/azure-template-stress/Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG REGISTRY="azsdkengsys.azurecr.io"
+FROM ${REGISTRY}/java/jdk-mariner-mvn:jdk11-latest as builder
+
+RUN yum -y update
+
+# Add necessary files to the image
+RUN mkdir /stress
+WORKDIR /stress
+ADD ./sdk/tools /stress/sdk/tools
+ADD ./eng /stress/eng
+ADD ./common /stress/common
+ADD ./sdk/parents /stress/sdk/parents
+ADD ./sdk/template /stress/sdk/template
+
+# Build dependencies and stress tests
+RUN --mount=type=cache,target=/root/.m2 \
```

Comment:
Comment here that the first build of this will be slow (because the cache isn't warmed up) but future builds should be faster?

Reply:
yep, packages will be downloaded incrementally when rebuilding the layer

> Cache mounts let you specify a persistent package cache to be used during builds. The persistent cache helps speed up build steps, especially steps that involve installing packages using a package manager. Having a persistent cache for packages means that even if you rebuild a layer, you only download new or changed packages.

https://docs.docker.com/build/guide/mounts/